### ISSUE TITLE
Remove workaround for noscript tag (issue with tags being escaped)

### ISSUE THREAD

**Issue by sidkshatriya** [Lullabot/amp-library#53]
In #22 there was a subtle workaround for `<noscript>` tags due to a problem with the `mastermind/html5` library versions < 2.2.0

Now that the upstream library has fixed its bug as of 2.2.0 we need to _remove_ this workaround. (See release 2.2.0 https://github.com/Masterminds/html5-php/releases/tag/2.2.0 . Also https://github.com/Masterminds/html5-php/issues/98 ).

_Not_ removing the workaround and running with 2.2.0 results in the noscript tag having html tags within `<noscript>` escaped:

i.e. instead of: 

`<noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style>`

We get:
`<noscript>&lt;style amp-boilerplate&gt;body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}&lt;/style&gt;</noscript>`

In other words with the later versions of the `masterminds/html5` library the workaround results in a html output issue.

**Comment by sidkshatriya** [comment]
This was a tricky one. However, the fix is not yet visible to the world even though I've pushed this out to master...

@karens @sirkitree 
I know one of you setup a github trigger from the amp-library to packagist. The trigger is supposed to update https://packagist.org/packages/lullabot/amp with the latest commit / tags whenever there is a push to the github repo. 

The trigger does not seem to be working (you will notice that dev-master is still stuck at 67faf76e914 on https://packagist.org/packages/lullabot/amp which is old)

Can you please have a look? (I don't have access to the `lullabot/amp` packagist account nor admin access to `lullabot/amp` github repo so I can't solve this issue...)

cc: @dsayswhat

**Comment by sidkshatriya** [comment]
The github triggers now work (pushes to master update the dev-master association on packagist and so forth) I don't know if this is something that was restored by us or was temporarily failing on packagist itself.

Anyways all is well now. Closing.

**Comment by sidkshatriya** [comment]
_Note for Drupal users_: 

If you have installed the `amp` module (and its dependencies such as the amp-library) by following the instructions on https://www.drupal.org/project/amp then you can get this fix (and any future changes) by

``` bash
# For Drupal 8
$ cd <drupal-8-docroot>
$ composer drupal-update

# For Drupal 7
$ cd <drupal-7-docroot>
$ drush composer-manager update
```

**Comment by sidkshatriya** [comment]
_Note for Drupal users (Part 2)_:

Lastly, if you're using a pure composer based distribution like Acquia Lightning ( https://www.drupal.org/project/lightning )

``` bash
# For Acquia Lightning
$ cd <apache-lightning-docroot>
$ cd ..
$ composer update
```
